index.html

```diff
@@ -2,19 +2,199 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
-    <!-- Add a link to your custom CSS here -->
-
-    <title>Responsive grid project</title>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+   <link rel="stylesheet" type="text/css" href="css/grid.css">
+   <!-- Add a link to your custom CSS here -->
+   <link rel="stylesheet" type="text/css" href="css/main.css">
+   <title>Responsive grid project</title>
 </head>
 
 <body>
+   <!-- SECTION-1 // NAVBAR ON TOP OF BACKGROUND IMAGE-->
+   <header>
+      <nav class="container">
+         <div class="row col-12 sm-col-12 md-col-12 lg-col-12 navbar-top">
```

Comment:
A row is always 100% width, so you don't need any of `col-12 sm-col-12 md-col-12 lg-col-12` on this line.

Some tips:
* An element is either a row (`row`), or a column (`col-*`) - it shouldn't be both at the same time.
* Rows (`row`) can only contain columns (one, or more than one)
* Column elements (`col-*`) are where you place all of your content.

Your structure should resemble the following:

```
.container
  .row
    .col-*
    .col-*
  .row
    .col-*
    .col-*
    ...
```